doc/sphinx/proof-engine/ltac.rst

```diff
@@ -52,127 +66,98 @@ the variable :token:`ident` matches any complex expression with (possible)
-                     : multimatch reverse goal with `context_rule` | ... | `context_rule` end
-                     : multimatch `ltac_expr` with `match_rule` | ... | `match_rule` end
-                     : abstract `atom`
-                     : abstract `atom` using `ident`
-                     : first [ `ltac_expr` | ... | `ltac_expr` ]
-                     : solve [ `ltac_expr` | ... | `ltac_expr` ]
-                     : idtac [ `message_token` ... `message_token`]
-                     : fail [`natural`] [`message_token` ... `message_token`]
-                     : gfail [`natural`] [`message_token` ... `message_token`]
-                     : fresh [ `component` … `component` ]
-                     : context `ident` [`term`]
-                     : eval `red_expr` in `term`
-                     : type of `term`
-                     : constr : `term`
-                     : uconstr : `term`
-                     : type_term `term`
-                     : numgoals
-                     : guard `test`
-                     : assert_fails `ltac_expr3`
-                     : assert_succeeds `ltac_expr3`
-                     : `tactic`
-                     : `qualid` `tacarg` ... `tacarg`
-                     : `atom`
-   atom              : `qualid`
-                     : ()
-                     : `int`
-                     : ( `ltac_expr` )
-   component : `string` | `qualid`
-   message_token     : `string` | `ident` | `int`
-   tacarg            : `qualid`
-                     : ()
-                     : ltac : `atom`
-                     : `term`
-   let_clause        : `ident` [`name` ... `name`] := `ltac_expr`
-   context_rule      : `context_hyp`, ..., `context_hyp` |- `cpattern` => `ltac_expr`
-                     : `cpattern` => `ltac_expr`
-                     : |- `cpattern` => `ltac_expr`
-                     : _ => `ltac_expr`
-   context_hyp       : `name` : `cpattern`
-                     : `name` := `cpattern` [: `cpattern`]
-   match_rule        : `cpattern` => `ltac_expr`
-                     : context [`ident`] [ `cpattern` ] => `ltac_expr`
-                     : _ => `ltac_expr`
-   test              : `int` = `int`
-                     : `int` (< | <= | > | >=) `int`
-   selector          : [`ident`]
-                     : `int`
-                     : (`int` | `int` - `int`), ..., (`int` | `int` - `int`)
-   toplevel_selector : `selector`
-                     : all
-                     : par
-                     : !
-
-.. productionlist:: coq
-   top              : [Local] Ltac `ltac_def` with ... with `ltac_def`
-   ltac_def         : `ident` [`ident` ... `ident`] := `ltac_expr`
-                    : `qualid` [`ident` ... `ident`] ::= `ltac_expr`
+   Syntactically, most :token:`tactic`\s are :production:`simple_tactic`\s.
+   The documentation also describes the following as :token:`tactic`\s because they can
```

Comment:
I don't think you need to use a token for tactic:
```suggestion
   Syntactically, most tactics are :production:`simple_tactic`\s.
   The documentation also describes the following as tactics because they can
```